selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/OpenOnGitHubTest.java

```diff
@@ -0,0 +1,160 @@
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.git;
+
+import static org.eclipse.che.commons.lang.NameGenerator.generate;
+import static org.eclipse.che.selenium.pageobject.CodenvyEditor.ContextMenuLocator.OPEN_ON_GITHUB;
+import static org.eclipse.che.selenium.pageobject.Wizard.TypeProject.BLANK;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.eclipse.che.selenium.core.client.TestGitHubRepository;
+import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.client.TestUserPreferencesServiceClient;
+import org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants;
+import org.eclipse.che.selenium.core.project.ProjectTemplates;
+import org.eclipse.che.selenium.core.user.DefaultTestUser;
+import org.eclipse.che.selenium.core.webdriver.SeleniumWebDriverHelper;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.CheTerminal;
+import org.eclipse.che.selenium.pageobject.CodenvyEditor;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Test "Open in Terminal" action. Will check action calling from context menu on selected project,
+ * folder and file
+ *
+ * @author Vitalii Parfonov
+ */
+public class OpenOnGitHubTest {
+  private static final String PROJECT_NAME = generate("project", 4);
+  private static final String PROJECT_WITHOUT_GIT_FOLDER = "projectWithoutGitCVS";
+  private static final String PATH_TO_EXPAND = "/src/main/java/che/eclipse/sample";
+  private String mainBrowserTabHandle;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private TestWorkspace workspace;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private Ide ide;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private ProjectExplorer projectExplorer;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private TestProjectServiceClient testProjectServiceClient;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private CheTerminal terminal;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private SeleniumWebDriverHelper seleniumWebDriverHelper;
+
+  @SuppressWarnings("unused")
+  @Inject
+  private SeleniumWebDriver seleniumWebDriver;
+
+  @Inject private CodenvyEditor editor;
+
+  @Inject private TestUserPreferencesServiceClient testUserPreferencesServiceClient;
+
+  @Inject private DefaultTestUser productUser;
+
+  @Inject private TestGitHubRepository testRepo;
+
+  @Inject
+  @Named("github.username")
+  private String gitHubUsername;
+
+  @Inject private org.eclipse.che.selenium.pageobject.git.Git git;
```

Comment:
You can shorten `org.eclipse.che.selenium.pageobject.git.Git` to `Git` if add FQN into import section.